django/utils/inspect.py

```diff
@@ -16,13 +16,18 @@ def _get_callable_parameters(meth_or_func):
     return _get_func_parameters(func, remove_first=is_method)
 
 
+arg_kinds = frozenset(
+    {
+        inspect.Parameter.POSITIONAL_ONLY,
```

Comment:
Though operations are serialised as kwargs, I thought it prudent to add positional-only args to this function.